tests/integration/cli/client_rpc_replicator_test.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package clitest
+
+/* WIP client rpc replicator getall is broken currently
```

Comment:
question: I'm curious as to what makes you say that? I manually tested the replicator commands on this branch and they seem to work.

Reply:
oops that's probably a mistake leftover comment